osi_trafficsign.proto

```diff
@@ -987,232 +987,3174 @@ message TrafficSign
+                // (StVO 1053-33, StVO 1053-37, StVO 1060-33).
+                //
+                // See e.g.: https://www.dvr.de/bilder/stvo/gt/1053-33.png
+                //           https://www.dvr.de/bilder/stvo/gt/1053-37.png
+                //           https://www.dvr.de/bilder/stvo/gt/1053-33.2.png
+                //
+                TYPE_WEIGHT = 6;
+
+                // Restriction of the validity of the traffic sign to wet /
+                // rainy conditions.
+                //
+                TYPE_RAIN = 7;
+
+                // Restriction of the validity of the traffic sign to foggy
+                // conditions.
+                // (StVO 1007-61).
+                //
+                // See e.g.:
+                //
+                TYPE_FOG = 8;
+
+                // Restriction of the validity of the traffic sign to snowy /
+                // icy conditions.
+                //
+                TYPE_SNOW = 9;
+
+                // Combined restriction of the validity of the traffic sign to
+                // rainy or snowy conditions denoted by a single supplementary
+                // sign (instead of two signs of \c #TYPE_RAIN and \c #TYPE_SNOW
+                // respectively).
+                //
+                TYPE_SNOW_RAIN = 10;
+
+                // Direction, left-pointing.
+                // (StVO 1000-10).
+                //
+                // See e.g.:
+                //
+                TYPE_LEFT_ARROW = 11;
+
+                // Direction, right-pointing.
+                // (StVO 1000-20).
+                //
+                // See e.g.:
+                //
+                TYPE_RIGHT_ARROW = 12;
+
+                // Direction of potential danger, left-pointing.
+                // (StVO 1000-11).
+                //
+                // See e.g.:
+                //
+                TYPE_LEFT_BEND_ARROW = 13;
+
+                // Direction of potential danger, right-pointing.
+                // (StVO 1000-21).
+                //
+                // See e.g.:
+                //
+                TYPE_RIGHT_BEND_ARROW = 14;
+
+                // Valid for heavy trucks.
+                // (StVO 1024-13).
+                //
+                // See e.g.:
+                //
+                TYPE_TRUCK = 15;
+
+                // Passing, only tractors only.
+                // (StVO 1024-17).
+                //
+                // See e.g.:
+                //
+                TYPE_TRACTORS_MAY_BE_PASSED = 16;
+
+                // Hazardous.
+                //
+                TYPE_HAZARDOUS = 17;
+
+                // Valid for vehicle with trailers.
+                //
+                TYPE_TRAILER = 18;
+
+                // Valid at night.
+                //
+                TYPE_NIGHT = 19;
+
+                // Zone.
+                //
+                TYPE_ZONE = 20;
+
+                // Stop 4 way.
+                //
```

Comment:
I think I already reviewed the following lines. Please check my first review.

Reply:
Yes, I was changing Allowed to Only and I missed most of the comments for supplementary signs.
I went over them again and made changes.